org.civicrm.reqangular/src/common/services/format-currency.service.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-env amd */
+
+define([
+  'common/modules/services',
+  'common/services/api',
+  'common/models/settings.model'
+], function (services) {
+  'use strict';
+
+  services.factory('FormatCurrencyService', FormatCurrencyService);
+
+  FormatCurrencyService.$inject = ['$log', '$q', 'Settings'];
+
+  function FormatCurrencyService ($log, $q, Settings) {
+    $log.debug('Service: FormatCurrencyService');
+
+    return {
+      format: format
+    };
+
+    /**
+    * Adds defined thousand and decimal separators in the given amount
+    *
+    * @param {string} amount
+    * @return {promise}
+    */
+    function format (amount) {
```

Comment:
problem is that now that the method returns an object that provides both the `formatted` and `unformatted` (is that even correct to call it that way?) amount, then the method has a kind of incorrect name

Reply:
Well, here is what actually happens:
1.  `formatted`:  Will hold the value with added thousand and decimal separators.
2.  `unformatted`: Will hold the value that is ready for calculation.

So, whatever the given amount, for both cases above we are formatting. So, i think its appropriate to call `format`. Does it makes sense?

cc @igorpavlov

Reply:
As I suggested above, `unformatted` might become `parsed`. 

1. `parsed`: Will hold the value that is ready for calculation.
2. `formatted`: Will hold the value with added thousand and decimal separators.

cc @AkA84